setup.py

```diff
@@ -29,6 +29,11 @@
         'flake8-simplify==0.19.3',
         'isort==5.10.1',
         'mccabe==0.7.0',
+        'mdformat==0.7.16',
```

Comment:
For a start, I'd like to limit it to just mdformat without any plugins, and add them later as adoption increases.

Reply:
As per usage instructions in https://mdformat.readthedocs.io/en/stable/users/installation_and_usage.html

"Note that GitHub’s Markdown renderer supports syntax extensions not included in the GFM specification. For full GitHub support do:"
```bash
pip install mdformat-gfm mdformat-frontmatter mdformat-footnote
```

Thought this might be useful to have, would you like me to drop this?